src/material-experimental/mdc-autocomplete/animations.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {
+  animate,
+  AnimationTriggerMetadata,
+  group,
+  state,
+  style,
+  transition,
+  trigger
+} from '@angular/animations';
+
+// Animation values come from
+// https://github.com/material-components/material-components-web/blob/master/packages/mdc-menu-surface/_mixins.scss
+// TODO(mmalerba): Ideally find a way to import the values from MDC's code.
+export const panelAnimation: AnimationTriggerMetadata = trigger('panelAnimation', [
+  state('hidden', style({})),
+  state('visible', style({})),
```

Comment:
I think you can omit these states completely, or at least set the `opacity: 0; transform: scale(0.8)` on the `hidden` state. Also we might want to add some styles for the `void` state.